package.json

```diff
@@ -0,0 +1,5 @@
+{
+  "dependencies": {
+    "dotenv": "^16.3.1"
```

Comment:
i find it unusual that we had to created a `package.json` at the root level of the application in order to set up our environment

@nghiapham1026 was this for debugging purposes? do other engineers need it to run the app?

Reply:
I do remember Nathan implementing this for the use of rerunning the application without having to update it every time. I do agree that we should discard this when the pull request occurs

Reply:
@9brian Correct me if I am wrong or your team is using this for some other purpose, but the environment variables should already be loadable from the root of the web application without the addition of the dotenv package.

Our team has these stored in `/apps/web/.env`, but other names (found in the .gitignore) for these .env files should also function similarly.

This is provided through [Next.JS](https://nextjs.org/docs/pages/building-your-application/configuring/environment-variables)

Reply:
@ConlynPattison Yeah, we had a rocky start with the environment variables, but they're up and running now. We're omitting this